test.clj

```diff
@@ -1,7 +1,22 @@
 ;;; font locking
 (ns clojure-mode.demo
-  (:require [clojure.something]
-            [something.s]))
+  (:require
+   [clojure.something]
+   [something.s]
+   [oneword]
+   ;; TODO mixedCase shouldn't be font-locked to clojure-interop-method-face
+   [mixedCase]
+   [CamelCase]))
+
+;; examples of valid namespace definitions
+(comment
+  (ns .validns)
```

Comment:
All of those should ideally be unit tests.

Reply:
I know, I know. Patience please :)

Reply:
Done. See the e5cc4d0 and a581bba.

Reply:
No problem. Just let me know when you get to this.